### ISSUE TITLE
skid

### ISSUE THREAD

**Issue by Officer32k** [bebeli555/CookieClient#12]
why do you skid modules? its painfully obvious just make your own its not that hard u dont know java or smth?

**Comment by bebeli555** [comment]
like 90% of the modules are made by me and the whole base is my own too. But some modules like ShulkerPreview is from another client as there's no reason to spend hours figuring out how to make it yourself. Every client has skidded something specially closed source clients probably skid way more as they can't be found out for it. I haven't given the best credits for modules that are from other clients as im pretty sure those clients skidded the module too.
